_rules/html-page-lang-valid-bf051a.md

```diff
@@ -19,83 +19,41 @@ authors:
 
 ## Applicability
 
-The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element with a `lang` and/or `xml:lang` attribute that is neither empty ("") nor only [whitespace](#whitespace).
+This rule applies to any [document element](https://dom.spec.whatwg.org/#document-element) if it is an `html` element that:
 
-**Note**: Documents embedded into other documents, such as through `iframe` or `object` elements are not applicable because they are not web pages according to the definition in WCAG.
+- has a [node document](https://dom.spec.whatwg.org/#concept-node-document) with a [content type](https://dom.spec.whatwg.org/#concept-document-content-type) of `text/html` and whose [browsing context](https://html.spec.whatwg.org/#browsing-context) is a [top-level browsing context](https://html.spec.whatwg.org/#top-level-browsing-context); and
+- has a `lang` attribute that is neither empty ("") nor only [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace)
 
 ## Expectation
 
-For each test target, the `lang` and `xml:lang` attributes have a [valid language subtag](#valid-language-subtag) if the attribute is neither empty ("") nor only [whitespace](#whitespace).
+For each test target, the `lang` attribute has a [valid language subtag](#valid-language-subtag).
 
 ## Assumptions
 
-_There are currently no assumptions_
+- The presence of a `lang` attribute is being used to comply to WCAG. This rule doesn't test if the attribute is needed to comply to WCAG.
 
 ## Accessibility Support
 
-There are known combinations of a popular operating system with browsers and assistive technologies that do not support the `lang` and `xml:lang` attributes.
-
-While HTML5 specification indicates that `xml:lang` attribute takes priority over `lang` attribute, certain assistive technologies prioritize `lang` over `xml:lang` instead.
+There are known combinations of a popular operating system with browsers and assistive technologies that do not support the `lang` attribute.
```

Comment:
I'm not sure this still rings true for just the `lang` attribute. I mean we experienced inconsistencies when testing with the `lang` and `xml:lang` attributes but I am not aware of a lack of support for the `lang` attribute.

Reply:
I tend to agree. Any thoughts on this, @jkodu, as one of the authors of this rule?

Reply:
I will remove it.